src/protocol.rs

```diff
@@ -450,6 +466,50 @@ mod tests {
     use std::collections::HashMap;
     use tracing::debug;
 
+    #[test]
+    fn participant_config_has_more_than_one_participant() {
+        let result = std::panic::catch_unwind(|| {
+            let mut rng = init_testing();
+            let QUORUM_SIZE = 1;
+            let _p = ParticipantConfig::random_quorum(QUORUM_SIZE, &mut rng);
```

Comment:
suggestion: this is up to you but you can also just pass 1 directly here. `QUORUM_SIZE` is only defined in the other function because we use it for other checks later.